src/main/scala/com/cognite/sdk/scala/common/auth.scala

```diff
@@ -40,3 +40,9 @@ final case class BearerTokenAuth(bearerToken: String, override val project: Opti
   def auth[U[_], T, S](r: RequestT[U, T, S]): RequestT[U, T, S] =
     r.header("Authorization", s"Bearer $bearerToken")
 }
+
+final case class AuthTicketAuth(authTicket: String, override val project: Option[String] = None)
```

Comment:
I'd prefer just `TicketAuth` unless you really want `AuthTicketAuth` for some reason (maybe that's more natural in the context, even if it looks weird to me because I haven't used those things?)

Reply:
TicketAuth is fine by me :+1: